components/tooltip/tooltip.js

```diff
@@ -0,0 +1,238 @@
+import { clearDismissible, setDismissible } from '../../helpers/dismissible.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodyCompactStyles } from '../typography/styles.js';
+import { getUniqueId } from '../../helpers/uniqueId.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+class Tooltip extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			delay: { type: Number },
+			disableFocusLock: { type: Boolean, attribute: 'disable-focus-lock' },
+			for: { type: String },
+			forceShow: { type: Boolean, attribute: 'force-show' },
+			showing: { type: Boolean, reflect: true, attribute: 'showing' },
+			state: { type: String, reflect: true }, /* Valid values are: 'info' and 'error' */
```

Comment:
Does `state` being info actually do anything, or is `error` the only value that matters? Would it make sense to just have a Boolean of `error` or something like that?

Reply:
I personally don't mind the state being a string -- it's the same as we did for `status-indicators`. It'll allow us to have other states in the future if we need them I guess?

Reply:
It was inspired by `state` in `status-indicators`. I added it because before it was entirely configurable with CSS variables if people needed different colors or styles. Now `state` gives a clear place to add other possible variants if people need them in the future.